src/main/java/wolox/training/controllers/BookController.java

```diff
@@ -105,7 +134,12 @@ public String greeting(@RequestParam(name="name", required=false, defaultValue="
      * @throws BookNotFoundException: When the book id that was passed doesn't belong to any book
      */
     @DeleteMapping("{id}")
-    public ResponseEntity<Void> delete(@PathVariable int id) throws BookNotFoundException{
+    @ApiOperation(value = "Deletes a specified user", response = User.class)
+    @ApiResponses(value = {
+            @ApiResponse(code = 204, message = "No Content"),
+            @ApiResponse(code = 404, message = "Not Found")
+    })
+    public ResponseEntity<Void> delete(@ApiParam(value = "id", type = "path", required = true, name = "id", example = "1") @PathVariable int id) throws BookNotFoundException{
```

Comment:
You must be careful with the line width, in wolox we usually do not exceed 120 characters, this can be configured from in IntelliJ

Reply:
Done!